src/bussinessLogic/Constants.java

```diff
@@ -0,0 +1,18 @@
+package bussinessLogic;
+
+import interfaces.ConstantsInterface;
+
+public class Constants implements ConstantsInterface {
```

Comment:
The constant class could be created without the interface, in this case.
All of her properties can be set as public constants.
In this case the class could look like the following.

```
public class Constants {
    /**
     * the number of letters in the alphabet
     */
    public static final int NUMBER_OF_LETTERS = 3;

    /** 
      * int for calculating how many times a letter is displayed in the evaluation sequence
      */
    public static final int ALPHABETMULTIPLIER = 1;
}
```

This way, there are fewer lines of code for those constants.

_Note_: In Java constants are declared all caps.

Reply:
:+1:

Reply:
One question: should I move the RANDOM constant from `ManageArray` to the `Constants` class or leave it there? (RANDOM is used only locally in  `ManageArray` class)

Reply:
It can stay in `ManageArray` class, for now.

Reply:
OK. Thanks!